code/main/media/mediaModule.brs

```diff
@@ -96,6 +96,14 @@ function _adb_MediaModule(configurationModule as object, edgeRequestQueue as obj
             mediaHit.tsObject = tsObject
             return mediaHit
         end sub,
+
+        dump: function() as object
+            return {
+                clientSessionId: m._sessionManager.getActiveClientSessionId(),
+                backendSessionId: m._sessionManager._getBackendSessionId(),
+                existActiveSession: m._sessionManager._existActiveSession(),
```

Comment:
if we think we need this:
can we update:
`_activeSessionExists()` or
`_hasActiveSession()`